booster-firejail.pro

```diff
@@ -0,0 +1,28 @@
+TEMPLATE = app
+
+TARGET = booster-firejail
+QT += concurrent
```

Comment:
Note seeing this either

Reply:
Ah, in the buildrequires... and this adds the linking

Reply:
Removed